mars/tensor/execution/tests/test_reduction_execute.py

```diff
@@ -267,8 +267,12 @@ def testArgReduction(self):
         self.assertTrue(np.array_equal(raw.argmin(axis=0),
                         self.executor.execute_tensor(arr.argmin(axis=0), concat=True)[0]))
 
-        raw = sps.random(20, 20, density=.1)
+        row = np.array([0, 1, 4, 9, 13, 19])
+        col = np.array([0, 1, 4, 9, 13, 19])
+        data = np.array([0, 1, 2, 3, 4, 5])
+        np.random.shuffle(data)
```

Comment:
Better to specify sps.random with format='lil', and ten set a random position with a bigger number, assert the argmax is equivalent to this position